src/main/java/ca/corefacility/bioinformatics/irida/config/services/IridaApiServicesConfig.java

```diff
@@ -288,7 +289,9 @@ private String getClasspathResourceBasename(Pattern pattern, Resource x) {
 
 	@Bean(name = "iridaFileStorageUtility")
 	public IridaFileStorageUtility iridaFileStorageUtility() {
```

Comment:
Can you add a comment that we're configuring both the `IridaFileStorageUtility` and `IridaFiles` in here?  I know it'll look different for the cloud impls later.

Reply:
Sure. Added in 987e491117e9ca0bced011205e6e3d7e201914a3